src/desert/_make.py

```diff
@@ -289,7 +289,8 @@ def field_for_schema(
 
     if field is None:
         nested = forward_reference or class_schema(typ)
-        field = marshmallow.fields.Nested(nested)
+        params = {k: v for k, v in metadata.items() if type(k) is str}
+        field = marshmallow.fields.Nested(nested, **params)
```

Comment:
Because the Nested class accepts `**kwargs` in its signature, we don't have to worry about extraneous params breaking anything downstream.